mscore/musescore.cpp

```diff
@@ -2967,6 +2967,12 @@ AboutBoxDialog::AboutBoxDialog()
       setWindowFlags(this->windowFlags() & ~Qt::WindowContextHelpButtonHint);
 
       copyRevisionButton->setIcon(*icons[int(Icons::copy_ICON)]);
+
+      copyrightLabel->setText(QString("<span style=\"font-size:10pt;\">%1</span>")
```

Comment:
yes, I was contemplating this, also for security reasons, a bogus translation could redirect to some entirely different place.
There are other places though where HTML is in the translations, like the save onlyin dialogs and some error messages